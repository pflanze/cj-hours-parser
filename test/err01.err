Error: can't process file "test/err01.txt"

Caused by:
    line 14 column 1: garbage after ']': " garble"
