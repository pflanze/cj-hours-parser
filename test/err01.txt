# this is a comment
[
this is a multiline
  comment
]
-12:44 test  some time

[[
some more commenting
]] 
#[and yet more]
[
blabla
] garble
13:27-18:44 test-again

