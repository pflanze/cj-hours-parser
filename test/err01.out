item=SingleLineComment(Owned(" this is a comment"))
    =SingleLineComment(Owned(" this is a comment"))
item=MultiLineComment([Owned("this is a multiline"), Inline("  comment")])
    =MultiLineComment([Owned("this is a multiline"), Inline("  comment")])
item=Other([Minus, Time(NT(12:44:00)), Rest(Inline("test  some time"))])
    =Other([Subitem(L(0, 6, 0), Minus), Subitem(L(0, 6, 1), Time(NT(12:44:00))), Subitem(L(0, 6, 7), Rest(Inline("test  some time")))])
item=Empty(1)
    =Empty(1)
item=MultiLineComment([Owned("some more commenting")])
    =MultiLineComment([Owned("some more commenting")])
item=SingleLineComment(Inline("[and yet more]"))
    =SingleLineComment(Inline("[and yet more]"))
